src/main/java/com/hivemq/extensions/google/cloud/pubsub/customizations/helloworld/MqttToPubSubHelloWorldTransformer.java

```diff
@@ -104,12 +104,12 @@ public void transformMqttToPubSub(
                     publishPacket.getPayload().ifPresent(builder::data);
                     outboundPubSubMessages.add(builder.build());
                 } catch (final Exception e) {
-                    LOG.error("Could not create a GCP PubSub message from MQTT message with topic '{}' because", mqttTopic, e);
+                    LOG.error("Could not create a Google Cloud PubSub message from MQTT message with topic '{}' because", mqttTopic, e);
```

Comment:
```suggestion
                    LOG.error("Could not create a Google Cloud Pub/Sub message from MQTT message with topic '{}' because", mqttTopic, e);
```